Create folder design with all mockups (#54)
KrispyTech/airneis: docs/design/mockups/web_mobile/categorypage.md

# Category Page Mockup

![Category Page Mockup](./images/categorypage.png)

> The category page is the first page containing the products you want to see or buy on the site or on your mobile phone.

Reviewer: ```suggestion
The category page is the first page containing the products you want to see or buy on our website or our mobile app.
```

> The first part of the mackup is an image containing the category title:

Reviewer: ```suggestion
The first part of the mock-up is an image containing the category title:
```

- To check that we are in the category we want.

> The second part of the mackup represents the product sheets:

Reviewer: ```suggestion
The second part of the mock-up represents the product sheets:
```
